src/Components/Home.js

```diff
@@ -2,21 +2,37 @@ import './Home.css';
 import { useState } from 'react';
 
 const Home = () => {
-    const [value, setValue] = useState('');
+    const initialValue = [];
+    const [value, setValue] = useState([]);
+    const [initial, setInitial] = useState(false);
```

Comment:
This state will not be required with Controlled Component.
i.e. create a state for individual todo and reset the state on it's addition to empty the input field.